GoudeauRampUp/Assets/Scripts/MoveP1.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MoveP1 : MonoBehaviour
+{
+
+    [SerializeField]
+    private float speed = 5;
```

Comment:
Get in the habit of writing f after your floats (ie 5f) - that makes them cleaner, not to mention often the code won't compile without it